zerver/webhooks/raygun/doc.md

```diff
@@ -0,0 +1,30 @@
+Raygun is an error and performance monitoring for web and mobile apps. Follow
```

Comment:
Make this `error and performance monitoring software`. :)

Reply:
okay!